packages/kbn-securitysolution-exceptions-common/api/find_exception_list/find_exception_list.gen.ts

```diff
@@ -0,0 +1,70 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+/*
+ * NOTICE: Do not edit this file manually.
+ * This file is automatically generated by the OpenAPI Generator, @kbn/openapi-generator.
+ *
+ * info:
+ *   title: Find exception lists API endpoint
+ *   version: 2023-10-31
+ */
+
+import { z } from 'zod';
+import { ArrayFromString } from '@kbn/zod-helpers';
+
+import { NonEmptyString } from '@kbn/openapi-common/schemas/primitives.gen';
+import { ExceptionNamespaceType, ExceptionList } from '../model/exception_list_common.gen';
+
+export type FindExceptionListsFilter = z.infer<typeof FindExceptionListsFilter>;
+export const FindExceptionListsFilter = NonEmptyString;
+
+export type FindExceptionListsRequestQuery = z.infer<typeof FindExceptionListsRequestQuery>;
+export const FindExceptionListsRequestQuery = z.object({
+  /** 
+      * Filters the returned results according to the value of the specified field.
+
+Uses the `so type.field name:field` value syntax, where `so type` can be:
+
+- `exception-list`: Specify a space-aware exception list.
+- `exception-list-agnostic`: Specify an exception list that is shared across spaces.
+ 
+      */
+  filter: FindExceptionListsFilter.optional(),
```

Comment:
in `packages/kbn-securitysolution-io-ts-list-types/src/common/filter/index.ts` non empty string for `filter` is not required